packages/actor-extract-links-extract-tree/lib/ActorExtractLinksTree.ts

```diff
@@ -42,21 +42,47 @@ export class ActorExtractLinksTree extends ActorExtractLinks {
         this.getTreeQuadsRawRelations(quad,
           currentNodeUrl,
           pageRelationNodes,
-          nodeLinks));
+          nodeLinks,
+          relationDescriptions));
 
       // Resolve to discovered links
-      metadata.on('end', () => {
+      metadata.on('end', async() => {
```

Comment:
Be careful with this: if `mediatorOptimizeLinkTraversal` would reject, then this promise rejection will be lost.
Instead of creating such an anonymous async function, I would keep it non-async, and work with `.then()` on the mediator (and then also `.catch()`)